src/app/@ansyn/ansyn/modules/menu-items/tools/components/measure-control/measure-control.component.ts

```diff
@@ -83,7 +84,8 @@ export class MeasureControlComponent implements OnInit, OnDestroy, IEntryCompone
 	toggleShowLayer() {
 		this.store$.dispatch(new UpdateMeasureDataOptionsAction({
 			mapId: this.mapId,
-			options: { isLayerShowed: !this.measureData.isLayerShowed }
+			options: { isLayerShowed: !this.measureData.isLayerShowed },
+			fromUI: true
```

Comment:
If you look at all the actions that we send, you will see that some actions are sent many times, from different places. The role of the flag `fromUI` is that we print to the log only relevant actions, and in particular actions that reflect actual actions of the user.
The action UpdateMeasureDataOptionsAction which we see here is sent from six places in the code. Only three of the six are the direct results of user action. The other three seem to be "just in case" actions, usually from effects.